packages/accounts-base/accounts_client.js

```diff
@@ -438,4 +453,24 @@ if (Package.blaze) {
   Package.blaze.Blaze.Template.registerHelper('loggingIn', function () {
     return Meteor.loggingIn();
   });
+
+  /**
+   * @global
+   * @name  loggingOut
+   * @isHelper true
+   * @summary Calls [Meteor.loggingOut()](#meteor_loggingout).
+   */
+  Package.blaze.Blaze.Template.registerHelper('loggingOut', function () {
```

Comment:
I realize there is already code in this file that assumes `Package.blaze` is defined, but I think I would prefer if we were a bit more defensive about checking whether `blaze` is installed, since it's only a weak dependency of `accounts-base`.